ooaclld/ooaclld/datatables.py

```diff
@@ -171,19 +171,19 @@ def col_defs(self):
             ]
         if self.language:
             return [
-                LinkCol(
+                IdCol(
                      self, 
                      "Parameter ID", 
                      sTitle="Feature ID", 
                      sClass="left", 
-                     model_col=OOAValue.parameter, 
-                     get_object = lambda i: i.valueset.parameter),
+                     model_col=OOAParameter.id,
+                     get_object=lambda i: i.valueset.parameter),
                 #IdCol(self, "Id", sTitle="Value ID", sClass="left"),
                 LinkCol(
                     self,
                     "Feature ID",
                     sTitle="Feature",
-                    model_col=OOAParameter.id,
+                    model_col=OOAParameter.name,
```

Comment:
I assume sorting and searching should use the parameter title, not the id, right?

Reply:
With the feature ID column now being the first in the table, sorting by ID should be the default. And yes, `IdCol` doesn't do infix search - but the feature title column now does that, and does that for the actual feature title text, which is displayed in the column. I always find it difficult, if the search query cannot readily be matched to its results.

Reply:
Is it possible to turn on infix searching for the IdCol, or to support this somehow? I am maybe projecting how we refer to features internally, but I would find it useful for the string "Noun" to filter for all IDs in "NounPoss", since our features are organized into domains.

Reply:
`IdCol` is just a small wrapper around `LinkCol`, see https://github.com/clld/clld/blob/b03984e8b88097c6f6247858207a573e070f85ff/src/clld/web/datatables/base.py#L238-L249
So you could just make your own `FeatureIdCol`:
```python
class FeatureIdCol(LinkCol):
    __kw__ = {'sClass': 'right', 'input_size': 'mini'}

    def get_attrs(self, item):
        return {'label': self.get_obj(item).id}
```

Reply:
Ah, thank you Robert!